### PR TITLE
Propose adding a tentative addition

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
I do tentative additions and find them helpful, especially for discovering unnecessary indirect dependencies. I suggest others do the same.

Thoughts?

**Comment by david-a-wheeler** [comment]
@SecurityCRob - maybe this could proposal could be mentioned in a BEST WG meeting? I think voting for approval seems like the cleanest way to add a whole new proposed item.

File: docs/Concise-Guide-for-Evaluating-Open-Source-Software.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ As a software developer, before using open source software (OSS) dependencies or
...
 8. **What is the software’s license?** Licenses are technically not security, but licenses can have a significant impact on security and sustainability. Ensure every component has a license, that it’s a widely-used [OSI license](https://opensource.org/licenses) if it’s OSS, and that it’s consistent with your intended use. Projects that won’t provide clear license information are less likely to follow other good practices that lead to secure software.
-9. **What are the results of code evaluation?** Even a brief review of software (by you, someone you hire, or someone else), along with recent changes to it, can give you some insight. Here are things to consider:
+9. **What happens on a tentative addition?** Try adding the dependency as a test, preferably in an isolated environment (in case it's malicious).
+   1. Does it exhibit malicious behaviour, e.g., does it attempt to exfiltrate sensitive data?
+   2. What new indirect dependencies does it add? Is they unexpectedly many or unnecessary? Ask components to reduce runtime dependencies (e.g., by having them differentiate between runtime, development time, and test time dependencies).
```
Comment (lehors) [inline docs/Concise-Guide-for-Evaluating-Open-Source-Software.md:42]: "Is they unexpectedly many or unnecessary?" doesn't look right and I'm not sure enough about what you meant to make a suggestion.

Comment (david-a-wheeler) [reply docs/Concise-Guide-for-Evaluating-Open-Source-Software.md:42]: Basically, if you suddenly double the number of dependencies, find yourself adding compilers & such in operations, etc. Not sure how to write that text though.

Comment (david-a-wheeler) [reply docs/Concise-Guide-for-Evaluating-Open-Source-Software.md:42]: I'll rewrite that text, hopefully it will become clearer.

**Comment by david-a-wheeler** [comment]
Note: the only *real* addition is a new point 9.

The old point 9 is renumbered to point 10. Markdown lists must align to the text *after* the number, so the sub-bullets must all get 1 extra space at the beginning. So this change looks larger than it really is.

**Review by SecurityCRob — APPROVED** [review]
+1

**Comment by balteravishay** [comment]
+1 looks good!

**Review by balteravishay — APPROVED** [review]

**Review by torgo — APPROVED** [review]

**Pull request merged**